pyanaconda/modules/payloads/payload/payload_base_interface.py

```diff
@@ -39,6 +39,14 @@ def connect_signals(self):
         self.watch_property("Sources", self.implementation.sources_changed)
         self.watch_property("RequiredSpace", self.implementation.required_space_changed)
 
+    @property
+    def Type(self) -> Str:
+        """Get type of this payload.
+
+        Possible types are values of the payload.base.constants.PayloadType enum
+        """
+        return self.implementation.type
```

Comment:
You are returning the enum value here instead of the string.

Reply:
Hmm and I was wondering why I don't getting string in my tests... I should have investigate more before accepting that.

Thanks for pointing that out.